src/bin/nydus-image/main.rs

```diff
@@ -74,13 +77,19 @@ pub struct OutputSerializer {
     blobs: Vec<String>,
     /// Performance trace info for current build.
     trace: serde_json::Map<String, serde_json::Value>,
+    /// Rafs version(v4, v5 or v6).
```

Comment:
`Rafs version(v4, v5 or v6).` -> `Rafs version(5 or 6).`

Reply:
In fact have no v4 for upstream codes. :)

Reply:
fine